src/sentry/static/sentry/app/views/organizationGroupDetails/groupDetails.jsx

```diff
@@ -130,8 +130,7 @@ const GroupDetails = createReactClass({
           );
           return;
         }
-
-        const project = this.props.project || ProjectsStore.getById(data.project.id);
+        const project = this.props.project || data.project;
```

Comment:
Does this component ever get passed `project` via props?

Reply:
hmm this description implies it would be passed project

https://github.com/getsentry/sentry/blob/d6e77ba9316ffc272a68a87dbc79f733582c0def/src/sentry/static/sentry/app/views/organizationGroupDetails/groupDetails.jsx#L30-L31

but I can't find any other usages of GroupDetails :/

Reply:
Ah the "project" version no longer exists, so it seems good to remove

Reply:
gotcha i'll go ahead and remove